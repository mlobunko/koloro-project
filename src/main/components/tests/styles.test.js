import React from "react";
import { shallow } from "enzyme";
import "enzyme-to-json";

import {
  StyledGradient,
  StyledImageGameOfThrones,
  StyledMain,
  StyledTextBlock
} from "../styles";

describe("StyledGradient", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<StyledGradient />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe("StyledImageGameOfThrones", () => {
  it("renders correctly, landscape true", () => {
    const landscape = true;
    const wrapper = shallow(<StyledImageGameOfThrones landscape={landscape} />);
    expect(wrapper).toMatchSnapshot();
  });

  it("renders correctly, landscape false", () => {
    const landscape = false;
    const wrapper = shallow(<StyledImageGameOfThrones landscape={landscape} />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe("StyledMain", () => {
  it("renders correctly, landscape true", () => {
    const landscape = true;
    const wrapper = shallow(<StyledMain landscape={landscape} />);
    expect(wrapper).toMatchSnapshot();
  });

  it("renders correctly, landscape false", () => {
    const landscape = false;
    const wrapper = shallow(<StyledMain landscape={landscape} />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe("StyledTextBlock", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<StyledTextBlock />);
    expect(wrapper).toMatchSnapshot();
  });
});
