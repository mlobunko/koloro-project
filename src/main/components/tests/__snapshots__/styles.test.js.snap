// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledGradient renders correctly 1`] = `
<div
  className="sc-bdVaJa cThvqa"
/>
`;

exports[`StyledImageGameOfThrones renders correctly, landscape false 1`] = `
<div
  className="sc-bwzfXH hquYfs"
/>
`;

exports[`StyledImageGameOfThrones renders correctly, landscape true 1`] = `
<div
  className="sc-bwzfXH gGTZba"
/>
`;

exports[`StyledMain renders correctly, landscape false 1`] = `
<div
  className="sc-htpNat hdBNbd"
/>
`;

exports[`StyledMain renders correctly, landscape true 1`] = `
<div
  className="sc-htpNat dgIRnh"
/>
`;

exports[`StyledTextBlock renders correctly 1`] = `
<div
  className="sc-bxivhb jcNPuI"
/>
`;
